changed tests into one test with params

rafalstanula opened the pull request.

rafalstanula committed:
changed tests into one test with params

rafalstanula committed:
Merge branch 'master' into rafalstanula

wojtekPi commented on src/test/java/StringCalculatorTest.java:
```diff
@@ -1,54 +1,76 @@
+//    }
+//
+//    @Test
+//    public void shouldReturnOneWhenOnePassed() {
```
Pleas remove commented code

wojtekPi commented on src/test/java/StringCalculatorTest.java:
```diff
@@ -1,54 +1,76 @@
+    private Object[][] paramsForTest(){
+        return new Object[][]{
+                {"", 0},
+                {"1",1},
```
nice tests!

wojtekPi reviewed (changes requested):
Please remove not used code.

rafalstanula committed:
deleted comments from StringCalculatorTest

rafalstanula committed:
Merge remote-tracking branch 'origin/rafalstanula' into rafalstanula

rafalstanula committed:
Created class PaymentServiceTest.
Created two tests for PaymentService.
Created class PaymentService.
Created method transferMoney inside class PaymentService.
